CheckManagement/LocalRepository.cs

```diff
@@ -26,11 +29,85 @@ public class LocalRepository : IRepository
         /// </summary>
         private string FolderPath { get; }
 
+        /// <summary>
+        /// A default repository name to use when one is not provided.
+        /// </summary>
+        private string _defaultName = MainConsts.LOCAL_REPO_NAME;
+
         public LocalRepository(string folderPath)
         {
             FolderPath = folderPath;
         }
 
+        /// <summary>
+        /// A unique identifier for the repository.
+        /// </summary>
+        public Guid Id { get { return new Guid(); } }
```

Comment:
Calling `new Guid()` creates a new Guid with the default value. So it will be the same each time it is called. But that is not obvious so I changed it to `default(Guid)` which is a little clearer about the value that will be returned.